.github/actions/prlinter/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "prlinter",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "keywords": [],
+  "license": "Apache-2.0",  
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
```

Comment:
That's a `jsii` extension, FYI :)

Reply:
You mean the `organization` key? Whoops :\

Reply:
Yup. It doesn't hurt though :)

Reply:
👍